src/deploy/functions/backend.ts

```diff
@@ -143,6 +157,8 @@ export function endpointTriggerType(endpoint: Endpoint): string {
     return endpoint.eventTrigger.eventType;
   } else if (isTaskQueueTriggered(endpoint)) {
     return "taskQueue";
+  } else if (isBlockingTriggered(endpoint)) {
+    return "blocking";
```

Comment:
endpoint.blockingTrigger.eventType. It's much more interesting to know this is a beforeUserCreated event than that it's some generic blocking function.